### PR TITLE
fix: do not remove import else block

### PR DESCRIPTION
**Pull request by CaselIT**

### CODE REVIEW

File: test_files/delayed_import_typing.expected.py [hunk 1]
```diff
@@ -0,0 +1,13 @@
+import typing
+
+if typing.TYPE_CHECKING:
+    import bmemcached  # noqa
```
Comment (zzzeek) [inline test_files/delayed_import_typing.expected.py]: one problem i was having was that it was re3moving "bmemcached" for some reason, couldnt figure that out.

can we take out the "# noqa" s here?  this block should proceed unaffected

**Comment by zzzeek** [comment]
i pushed that change.  it works great

File: test_files/delayed_import_typing.expected.py (hunk 1 above)
Comment (zzzeek) [reply test_files/delayed_import_typing.expected.py]: just did it

**Comment by CaselIT** [comment]
let's merge and release then!

**Comment by CaselIT** [comment]
https://github.com/sqlalchemyorg/zimports/releases/tag/v0.4.1

also on pypi

**Comment by zzzeek** [comment]
it all works huih?  great!  :)

**Comment by CaselIT** [comment]
Seems so. We can fix this again if we have other issues :)

**Pull request merged**